source/guides/tooling/IDE-integration.md

```diff
@@ -19,6 +19,7 @@ There are many third-party IDE extensions and plugins to help integrate your IDE
 Compatible with IntelliJ IDEA, AppCode, CLion, GoLand, PhpStorm, PyCharm, Rider, RubyMine, and WebStorm.
 
 - {% url "Intellij-Cypress" https://plugins.jetbrains.com/plugin/13819-intellij-cypress %}: Integrates Cypress under the common Intellij test framework.
+- {% url "Cypress-Pro" https://plugins.jetbrains.com/plugin/13987-cypress-pro %}: An improved version of Intellij-Cypress plugin with debugging from the IDE, advanced autocomplete and other features.
```

Comment:
When you say "improved," does this mean that I still need `Intellij-Cypress` with this plugin? Or is this replacing the existing one?

Reply:
CypressPro is replacing the original Intellij-Cypress plugin. So it has all the functionality form the latter and adds some additional features. 
Do you think this sentence should be rephrased?

Reply:
I think it's confusing because there's two plugins now recommended here for IntelliJ. Can we mark this one as the recommended one somehow or even delete the reference to the original `Intellij-Cypress` plugin?

Reply:
Okay, I deleted the `Intellij-Cypress` plugin reference.